schemas/com.snowplowanalytics.snowplow.storage/postgresql_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Snowplow PostgreSQL storage configuration",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.storage",
+        "name": "postgresql_config",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "host": {
+            "type": "string"
+        },
+        "database": {
+            "type": "string"
+        },
+        "port": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 65535
+        },
+        "sslMode": {
+            "type": "string",
+            "enum": ["disable", "require", "verify-ca", "verify-full"]
```

Comment:
Yes, I definitely do. These are constants in some sense. And we're doing so in many other places: sql/api query enrichment, currency conversion, application error, factotum, etc. It would be a good rule to choose capital letters enums if you don't have specific reasons to not do so.